app/forms/respondents/respondent_form.rb

```diff
@@ -51,6 +47,16 @@ def initialize(*args)
 
     private
 
+    def police_notified_presence
+      return if police_notified.blank?
+
+      value = __send__("police_notified_details_#{police_notified}".to_sym)
```

Comment:
I think this still works without the need for `.to_sym`

Reply:
done!